### DISCUSSION TITLE
How to set useRowSelect as false

### DISCUSSION THREAD

**Discussion by raggya** [TanStack/table#3373] [Q&A]
Hi All,

I'm trying to create a re-usable table component with row selection functionality using the below code reference from the API.

https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/row-selection-and-pagination

The component I'm creating will take data and columns as props but will have the ability to turn toggle features. 
So if table 1 has the feature to select rows, but table two doesn't have the feature, how to turn off useRowSelect for the entire table?
I could understand other features like useFilters and useSortBy can be disabled per column by setting the disable property.

**Comment by narendradharajiya** [comment]
Well, one hacky way is keep useRowSelect for both the case.

give ID to selection column which is showing the checkbox for selection.
For example :
`id: 'selection'`

Then based on condition you can hide the column using `hiddenColumns` parameter.
